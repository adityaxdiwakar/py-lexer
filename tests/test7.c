int main() {
    int a = 5;
    float b = 6.7;
    while (a < 10) {
        a += 3;
    }
    return 0;
}
