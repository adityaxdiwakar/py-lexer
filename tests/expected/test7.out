INT_WORD
IDENT(main)
LPAREN
RPAREN
LBRACK
INT_WORD
IDENT(a)
EQUALS
INT(5)
SEMI
FLOAT_WORD
IDENT(b)
EQUALS
FLOAT(6.7)
SEMI
WHILE
LPAREN
IDENT(a)
LT
INT(10)
RPAREN
LBRACK
IDENT(a)
PLUS
EQUALS
INT(3)
SEMI
RBRACK
RETURN
INT(0)
SEMI
RBRACK
